filer/utils/pil_exif.py

```diff
@@ -19,6 +19,9 @@ def get_exif(im):
 
 
 def get_exif_for_file(file_obj):
+    storage = getattr(file_obj, 'storage', None)
```

Comment:
when can file_obj.storage be none?

Reply:
Maybe never. But in fact, I don't know.
file_obj may have always a storage attribute, but I'm not sure again.
file_obj is always an MultiStorageFileField or it can be any object with 'name' attribute ?

My proposition is the more flexible as possible preserving compatibility.

Reply:
I think we should not handle a case unless we're sure it can happen, otherwise we will never know if it can, thus I suggest to do ``file_obj.storage directly``, if test fails because of it then we can keep ``getattr`` and add a comment. /cc @yakky